src/main/java/core/basesyntax/service/interfaces/AgeValidator.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.service.interfaces;
+
+public interface AgeValidator {
+    boolean isValid(Integer age);
+}
```

Comment:
Create only one service for validation without interface, just in class ValidationService